neuron/src/lib/Neuron/Zettelkasten/Graph.hs

```diff
@@ -45,11 +45,14 @@ backlinkForest conn z =
     . G.dfsForestBackwards z
     . G.induceOnEdge (== Just conn)
 
-backlinks :: Connection -> Zettel -> ZettelGraph -> [Zettel]
+backlinks :: Maybe Connection -> Zettel -> ZettelGraph -> [Zettel]
 backlinks conn z g =
-  filter (not . branches) $ G.preSet z $ G.induceOnEdge (== Just conn) g
+  filter (not . branches) $ G.preSet z $ G.induceOnEdge matcher g
   where
     branches bz = G.hasEdge g z bz
+    matcher = case conn of
+      Nothing -> isJust
+      Just _ -> (== conn)
```

Comment:
Rename and replace with:

```haskell
includeConn = maybe isJust (==) conn
```

Reply:
This change has been applied in the latest commit.